defaults/test_default.yaml

```diff
@@ -71,6 +71,7 @@ email_subject_postfix: ''
 collect_logs: false
 
 hinted_handoff: 'disabled'
+parallel_node_operations: true  # supported from Scylla 6.0
```

Comment:
Making it to be default looks fragile approach as of now.
"master" SCT gets used not only for latest Scylla versions.
Personally I use lots of different Scylla versions with master SCT.

Can we make it be implicit by checking scylla version?

Reply:
I agree, checking scylla version and/or feature enabled should be the better/safer approach.
But, we may still want to have parameter to control it (even if supported by the version/feature).